aspnetcore/index.yml

```diff
@@ -89,15 +65,25 @@ conceptualContent:
         - url: web-api/action-return-types.md
           itemType: concept
           text: "Controller action return types"
-        - url: web-api/advanced/formatting.md
-          itemType: concept
-          text: "Format response data"
-        - url: web-api/handle-errors.md
+        - url: fundamentals/apis.md
+          itemType: overview
+          text: "See more"
```

Comment:
Thanks.  Since we didn't have a landing page just for HTTP API apps, the overview was the only place I could point to as a "See more" starting place.  I hate to create landing pages for each area to go to as the "See more" since I think sub area landing pages will not really get very much traffic overall.  Thoughts?

I will move the overview (which was the "See more" link) above the tutorial and omit the return types link, but there will also no longer be a "See more".

Reply:
I don't think a see more link is essential to have in the hub, as the overview and tutorial will both have see also sections.